app/src/main/java/com/infomaniak/drive/MainApplication.kt

```diff
@@ -77,7 +79,8 @@ class MainApplication : Application(), ImageLoaderFactory, DefaultLifecycleObser
 
         ProcessLifecycleOwner.get().lifecycle.addObserver(this)
 
-        AppCompatDelegate.setDefaultNightMode(UiSettings(this).nightMode)
+        val uiSettings = UiSettings(this)
+        AppCompatDelegate.setDefaultNightMode(uiSettings.nightMode)
```

Comment:
Was that really necessary? Creating a `val` just for that is a bit too much?

Reply:
It's used later in the same function, lines 139 & 140.

Reply:
Maybe this function is a little too long 🤔